tools/tests/com.liferay.ide.gradle.core.tests/src/com/liferay/ide/gradle/core/tests/FindDependenceVisitor.java

```diff
@@ -0,0 +1,51 @@
+package com.liferay.ide.gradle.core.tests;
+
+
+import org.codehaus.groovy.ast.CodeVisitorSupport;
+import org.codehaus.groovy.ast.expr.ClosureExpression;
+import org.codehaus.groovy.ast.expr.MethodCallExpression;
+
+/**
+ * @author Lovett Li
+ */
+public class FindDependenceVisitor extends CodeVisitorSupport
```

Comment:
The classes for FindDependenceVisitor and GradleDependenceParser should be in the gradle.core plugin, not in gradle.core.tests